Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -200,15 +198,7 @@ void GeneralConfigPane::OnThrottlerChoiceChanged(wxCommandEvent& event)
 
 void GeneralConfigPane::OnCPUEngineRadioBoxChanged(wxCommandEvent& event)
 {
-  const int selection = m_cpu_engine_radiobox->GetSelection();
-
-  if (main_frame->g_pCodeWindow)
-  {
-    bool using_interp = (SConfig::GetInstance().iCPUCore == PowerPC::CORE_INTERPRETER);
-    main_frame->g_pCodeWindow->GetMenuBar()->Check(IDM_INTERPRETER, using_interp);
-  }
-
-  SConfig::GetInstance().iCPUCore = cpu_cores[selection].CPUid;
+  SConfig::GetInstance().iCPUCore = cpu_cores.at(event.GetSelection()).CPUid;
```

Comment:
`cpu_cores` is the only member left in `GeneralConfigPane` that doesn't match our naming conventions (starting with a `m_`) - do you think it would be appropriate to take care of that here too (unless you have other plans for it)?

OTOH `CCodeWindow` could use some love too (especially since its all over the place, with the prefix, with lower-case and upper-case members); but the members there are not touched in the rest of the changes. Longer term, we should strive to make those consistent; at the same time it becomes a little out of scope for this PR though.